features/expectations/uri.feature

```diff
@@ -1,15 +1,37 @@
-@stable
+@javascript @stable
 Feature: URI
 
-  Background:
+  Scenario: Matching HTTP message URI
```

Comment:
🤔 I wonder if we can use "request" instead of "HTTP message" (everywhere) in this case as I don't think URI will ever get validated on responses

Reply:
We can change scenario names as we want, but steps vocabulary would need to be reflected in Cucumber steps definitions in `gavel.js`. Isn't a problem per say, we could do it.

Reply:
During the last few days while I was working with `gavel-spec`, I thought about creating a separate issue to align vocabulary. It's quite different from feature to feature, and apart from being aligned, it could also influence the way steps definitions are implemented (less custom code, parametric steps).

Reply:
Good idea. Also when we're writing the docs, we should use the same terms.